src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -77,7 +91,12 @@ const ActiveState = ({
                     );
                 case 'task':
                     return (
-                        <li key={type + id} className="bcs-activity-feed-task-new" data-testid="task">
+                        <li
+                            key={type + id}
+                            className={classNames('bcs-activity-feed-task-new', { 'bcs-is-focused': isFocused })}
+                            data-testid="task"
+                            ref={refValue}
```

Comment:
Rather than passing refs up to the parent, can we pass something like `isActive` to the child components and allow them to scroll themselves into view? Similar to how [version items](https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/versions/VersionsItemButton.js) work?

Reply:
We decided to defer this change to a followup PR. May involve creating a reusable component like `<ScrollFocusable>` or similar as well.